### ISSUE TITLE
Add tests for ExportFullViewingKeyRequest

### ISSUE THREAD

**Issue by grod220** [penumbra-zone/web#414]

**Comment by hdevalence** [comment]
We shouldn't support this in the extension at all, the viewing key should never leave the extension.

**Comment by grod220** [comment]
I think we originally implemented this assuming the view service would call the custody service for it. But in the end, you can't initialize the view service without a fvk, so it can just be passed in in the beginning. Will open a PR to remove.

**Comment by hdevalence** [comment]
Yeah, the bigger risk IMO is that if we add it, we might later expose it. I think until we have a reason to feel otherwise, we should really strongly avoid creating any workflows that rely on users managing viewing keys, and should treat them as tantamount to seed phrases. We want to make it difficult for third parties to build tools that demand viewing keys (= irrevocable access forever) and steer them to using selective disclosure (chains of `TransactionPerspective`s)
